opentelemetry-api/tests/configuration/test_configuration.py

```diff
@@ -119,15 +119,12 @@ def test_boolean(self) -> None:
         },
     )
     def test_integer(self) -> None:
-        self.assertEqual(
-            Configuration().POSITIVE_INTEGER, 123
-        )  # pylint: disable=no-member
-        self.assertEqual(
-            Configuration().NEGATIVE_INTEGER, -123
-        )  # pylint: disable=no-member
-        self.assertEqual(
-            Configuration().NON_INTEGER, "-12z3"
-        )  # pylint: disable=no-member
+        # pylint: disable=no-member
+        self.assertTrue(isinstance(Configuration().POSITIVE_INTEGER, int))
+        self.assertEqual(Configuration().POSITIVE_INTEGER, 123)
+        self.assertTrue(isinstance(Configuration().NEGATIVE_INTEGER, int))
+        self.assertEqual(Configuration().NEGATIVE_INTEGER, -123)
+        self.assertEqual(Configuration().NON_INTEGER, "-12z3")
```

Comment:
Curious how these tests passed before this change

Reply:
Before, we didn't have tests that checked `type` (added them here with `isinstance`). And this:

```python
Python 3.8.3 (default, Jun 29 2020, 18:03:36) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.16.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: 1 == 1.0
Out[1]: True

In [2]: 
```

Reply:
Ahhhhh right, that makes sense now.